src/Form/composer.json

```diff
@@ -22,7 +22,7 @@
     },
     "require": {
         "php": ">=5.6.0",
-        "cakephp/event": "^3.0.0",
+        "cakephp/event": "^3.5.0",
         "cakephp/validation": "^3.0.0"
```

Comment:
This needs to be updated as well.
https://github.com/cakephp/cakephp/blob/master/src/Validation/ValidatorAwareInterface.php#L12